Library/Formula/ledger.rb

```diff
@@ -68,7 +68,9 @@ def install
 
     args += opts
 
+    system "./acprep", flavor, "make", *args
     system "./acprep", flavor, "make", "doc", *args
+    system "./acprep", flavor, "make", "check", *args if build.with? "check"
```

Comment:
@mikemcquaid Requested that `make check` should be run and be turned off by the `without-check` option. Unfortunately `make check` randomly fails I'm working on a fix in the upstream repository, until then `make check` should therefore not be run at all and I've removed the line.